src/modules/organizations/components/edit-details.jsx

```diff
@@ -2,22 +2,51 @@ import React from 'react';
 
 import { organizationShape } from '../model';
 
-const EditDetails = ({ organization }) => {
+import bindInput from '../../common/containers/bind-input';
+
+const EditDetails = ({ organization, updateOrganization }) => {
   if (!organization) {
     return (
       <div>Loading...</div>
     );
   }
 
+  const fields = {};
+
+  const collect = () => {
+    const result = {};
+    Object.keys(fields).forEach((fieldName) => {
+      result[fieldName] = fields[fieldName].value;
+    });
+    return result;
+  };
+
+  const doUpdate = () => {
+    const patch = collect();
+    updateOrganization(patch);
+  };
+
+  const NameInput = bindInput(organization.display_name)(<input type="text" />);
+  const DescriptionInput = bindInput(organization.description)(<input style={{ width: '500px' }} />);
+  const LanguageInput = bindInput(organization.primary_language || '')(<input />);
+
   return (
     <div>
       <h2>{organization.display_name}</h2>
-      <p><small>{organization.description}</small></p>
-      <p>You are editing this organization</p>
+      <p><small>You are editing this organization</small></p>
+      Name: <NameInput withRef={(n) => { fields.display_name = n; }} />
+      <br />
+      Description: <DescriptionInput withRef={(n) => { fields.description = n; }} />
+      <br />
+      Primary Language: <LanguageInput withRef={(n) => { fields.primary_language = n; }} />
```

Comment:
I don't think this should be directly user settable. PFE is using counterpart to set the language on the create action. Not sure if we should use counterpart here, but it can be set with `navigator.language"